src/Symfony/Component/Intl/Tests/DateFormatter/Verification/IntlDateFormatterTest.php

```diff
@@ -45,6 +45,8 @@ public function testFormatWithTimezoneFromEnvironmentVariable()
 
     protected function getDateFormatter($locale, $datetype, $timetype, $timezone = null, $calendar = IntlDateFormatter::GREGORIAN, $pattern = null)
     {
+        IntlTestHelper::requireFullIntl($this, '55.1');
```

Comment:
What's the goal of this change? 

Currently tests need to be run against the specific version of ICU (the one that matches the stub version). Some tests will fail with ICU 55.1 if stubs are at 57.1 due to differences introduced between versions. Example:

```
1) Symfony\Component\Intl\Tests\NumberFormatter\NumberFormatterTest::testFormatCurrencyWithCurrencyStyleCostaRicanColonsRounding with data set #0 (100, 'CRC',
 'CRC', '%s100.00', 0, SplObjectStorage Object ())
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'CRC100.00'
+'CRC100'
```